renderer/src/components/WalletTransactionsHistory.tsx

```diff
@@ -0,0 +1,142 @@
+import { FC } from 'react'
+import { FILTransaction } from '../typings'
+import dayjs from 'dayjs'
+import { ReactComponent as IncomeIcon } from '../assets/img/icons/income.svg'
+import { ReactComponent as OutcomeIcon } from '../assets/img/icons/outcome.svg'
+import { ReactComponent as ExternalLinkIcon } from '../assets/img/icons/external.svg'
+import { ReactComponent as CafeIcon } from '../assets/img/icons/cafe.svg'
+import { ReactComponent as TransferIcon } from '../assets/img/icons/transfer.svg'
+import { ReactComponent as WalletIcon } from '../assets/img/icons/wallet.svg'
+import WalletTransactoinStatusWidget from './WalletTransactionStatusWidget'
+import { brownseTransactionTracker } from '../lib/station-config'
+
+interface WalletTransactionsHistoryProps {
+  allTransactions: FILTransaction[] | [],
+  latestTransaction: FILTransaction | undefined
+}
+
+const WalletTransactionsHistory: FC<WalletTransactionsHistoryProps> = ({ allTransactions = [], latestTransaction = undefined }) => {
+  const allTransactionsExcludingCurrent = allTransactions.filter((t) => (t.timestamp !== latestTransaction?.timestamp))
+
+  return (
+    <>
+      {allTransactions.length > 0
+        ? <>
+          {latestTransaction && <RecentTransaction key={latestTransaction.timestamp} transaction={latestTransaction} />}
+          <div className='pt-8'>
+            <p className="px-8 mb-2 w-fit text-body-3xs text-black opacity-80 uppercase">WALLET HISTORY</p>
+            {allTransactionsExcludingCurrent.map(
+              (transaction) => <Transaction key={transaction.timestamp} transaction={transaction} />)}
+          </div>
+        </>
+        : <div className='flex flex-col gap-3 px-6 pt-12'>
```

Comment:
Same comment as above regarding extracting the markup into `render` function calls